firebaseFunctions/src/helpers.ts

```diff
@@ -159,6 +163,99 @@ export const updateDealUpdatesCollection = (dealId: string): void => {
   });
 };
 
+/**
+ * DeepDAO api returns mixed avatar URL, which in some cases contains
+ * the full URL and in others return only the file name.
+ * This function unify urls to always contain the full URL path.
+ * @param url string
```

Comment:
> The API returns a mix of full URL (https://.../avatar.png) and only file name (avatar.png)

([quote source](https://github.com/PrimeDAO/prime-deals-dapp/pull/1135/files#r949043245))

Can you add some concrete urls please?
It makes it easier in the future to understand how we might need to change the functions, if the url changes

Reply:
Second point:

> ar3. Unfortunately, there's no documentation for URLs (or anything).

([quote sources](https://github.com/PrimeDAO/prime-deals-dapp/pull/1135/files#r949043245))

Can we ask them how this might change in the future?
Or "How stable will the avatar url be from their prediction?"

Maybe we don't need to ask them, but then could you add few instructions of how we can detect and solve the case of changing avatar-storage url?

Reply:
I’m not sure we need to be worried about it. If the base url change most likely the’ll have to add it into the logo url anyway.